src/main/java/lotto/model/LottoTickets.java

```diff
@@ -4,15 +4,28 @@
 import java.util.List;
 
 public class LottoTickets {
```

Comment:
아래 static 변수를 추가하고
```suggestion
private static final List<LottoGenerator> lottoGenerators = Arrays.asList(new AutoTicket(), new ManualTicket());
```
새로운 LottoGenerator가 추가되도 combineTickets 메서드를 수정하지 않도록 개선해볼까요?
수동, 자동 각각 생성하는 코드가 중복이니 이 부분을 제거할 방법을 찾아보면 개선 가능할거에요